pkg/report/report.go

```diff
@@ -81,19 +83,23 @@ type Report interface {
 	Stats() <-chan Stats
 }
 
-func NewReport(precision string) Report { return newReport(precision) }
+func NewReport(precision, reportType string, generatePerfReport bool) Report {
```

Comment:
Should it be more like `name` for `reportName` instead of `reportType` ?

Reply:
Or `benchmarkType` if it's more coherent..

Reply:
`name` as this is name of report, not a type.